src/fides/api/alembic/migrations/versions/cb344673f633_add_s3_connection_type.py

```diff
@@ -0,0 +1,53 @@
+"""add s3 connection type
+
+Revision ID: cb344673f633
+Revises: 5fe01e730171
+Create Date: 2024-05-31 20:46:08.829330
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "cb344673f633"
+down_revision = "5fe01e730171"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add 's3' to ConnectionType enum
+    op.execute("alter type connectiontype rename to connectiontype_old")
+    op.execute(
+        "create type connectiontype as enum('mongodb', 'mysql', 'https', 'snowflake', "
+        "'redshift', 'mssql', 'mariadb', 'bigquery', 'saas', 'manual', 'manual_webhook', "
+        "'timescale', 'fides', 'sovrn', 'attentive', 'dynamodb', 'postgres',"
+        "'generic_consent_email', 'generic_erasure_email', 's3')"
```

Comment:
this is the most important place to make sure we add scylla, others will get flagged with merge conflicts

Reply:
thanks for calling out, i reconciled in https://github.com/ethyca/fides/pull/4930/commits/11d9b63e63c467c1edad4e318e8463ca6051adec